src/routes/index.js

```diff
@@ -11,28 +12,35 @@ const envConfig = require('../config/env');
 const userService = require('../service/user');
 const fileService = require('../service/file');
 const fileUtil = require('../util/file');
+const { getData } = require('../service/aws');
+const { S3_BUCKET_ENABLED } = require('../config/env');
```

Comment:
envConfig is already required above.
Either update the existing require statement to include the specific variables used in this file, or reference the existing envConfig.